### PR TITLE
Glue tutorial

### PR DESCRIPTION
**Pull request by ekmb**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -104,6 +104,9 @@ To run tutorials:
    * - NLP
      - Token Classification (Named Entity Recognition)
      - `Token_Classification_Named_Entity_Recognition_tutorial.ipynb <https://colab.research.google.com/github/NVIDIA/NeMo/blob/main/tutorials/nlp/Token_Classification_Named_Entity_Recognition_tutorial.ipynb>`_
```
Comment (yzhang123) [inline README.rst:106]: could you delete the _tutorial suffix for consistency with rest of naming?

File: README.rst [hunk 2]
```diff
@@ -104,6 +104,9 @@ To run tutorials:
...
+   * - NLP
+     - GLUE Benchmark
+     - `GLUE_Benchmark_tutorial.ipynb <https://colab.research.google.com/github/NVIDIA/NeMo/blob/main/tutorials/nlp/GLUE_Benchmark_tutorial.ipynb>`_
```
Comment (yzhang123) [inline README.rst:109]: could you delete the _tutorial suffix for consistency with rest of naming?

File: tutorials/nlp/GLUE_Benchmark_tutorial.ipynb [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
+        "\n",
+        "* MNLI - [The Multi-Genre Natural Language Inference Corpus](https://cims.nyu.edu/~sbowman/multinli/multinli_0.9.pdf) is a crowdsourced collection of sentence pairs with textual entailment annotations. Given a premise sentence and a hypothesis sentence, the task is to predict whether the premise entails the hypothesis (entailment), contradicts the hypothesis (contradiction), or neither (neutral). The task has the matched (in-domain) and mismatched (cross-domain) sections.\n",
+        "* QNLI - [The Stanford Question Answering Dataset](https://nlp.stanford.edu/pubs/rajpurkar2016squad.pdf) is a question-answering dataset consisting of question-paragraph pairs, where one of the sentences in the paragraph (drawn from Wikipedia) contains the answer to the corresponding question. The task is to determine whether the context sentence contains the answer to the question.\n",
+        "* RTE The Recognizing Textual Entailment (RTE) datasets come from a series of annual [textual entailment challenges](https://aclweb.org/aclwiki/Recognizing_Textual_Entailment). The task is to determine whether the second sentence is the entailment of the first one or not.\n",
```
Comment (yzhang123) [inline tutorials/nlp/GLUE_Benchmark_tutorial.ipynb:103]: no links to this and following datasets?

File: tutorials/nlp/GLUE_Benchmark_tutorial.ipynb [hunk 4]
```diff
@@ -0,0 +1,540 @@
...
+        "* WNLI - The Winograd Schema Challenge\n",
+        "is a reading comprehension task in which a system must read a sentence with a pronoun and select the referent of that pronoun from a list of choices (Hector Levesque, Ernest Davis, and Leora Morgenstern. The winograd schema challenge. In Thirteenth International Conference on the Principles of Knowledge Representation and Reasoning. 2012).\n",
+        "\n",
+        "All tasks are classification tasks, except for the STS-B task which is a regression task. All classification tasks are 2-class problems, except for the MNLI task which has 3-class.\n",
```
Comment (yzhang123) [inline tutorials/nlp/GLUE_Benchmark_tutorial.ipynb:107]: has "3 classes"

File: tutorials/nlp/GLUE_Benchmark_tutorial.ipynb [hunk 5]
```diff
@@ -0,0 +1,540 @@
...
+      "metadata": {
+        "id": "ZnuziSwJ1yEB",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Datasets\n",
+        "\n",
+        "To download the GLUE data, you can use [this script](https://gist.githubusercontent.com/W4ngatang/60c2bdb54d156a41194446737ce03e2e/raw/17b8dd0d724281ed7c3b2aeeda662b92809aadd5/download_glue_data.py) and then run:\n",
+        "\n",
+        "`python download_glue_data.py`\n",
```
Comment (yzhang123) [inline tutorials/nlp/GLUE_Benchmark_tutorial.ipynb:123]: add link?

File: tutorials/nlp/GLUE_Benchmark_tutorial.ipynb [hunk 6]
```diff
@@ -0,0 +1,540 @@
...
+        "\n",
+        "use `--tasks TASK` if datasets for only selected GLUE tasks are needed\n",
+        "\n",
+        "After running the above commands, you will have a folder `glue_data` with data folders for every GLUE task. For example, data for MRPC task would be under glue_data/MRPC.\n",
+        "\n",
+        "This tutorial and `examples/nlp/glue_benchmark/glue_benchmark.py` work with all GLUE tasks without any modifications. For this tutorial, we are going to use MRPC task.\n",
```
Comment (yzhang123) [inline tutorials/nlp/GLUE_Benchmark_tutorial.ipynb:129]: add link?

File: tutorials/nlp/GLUE_Benchmark_tutorial.ipynb [hunk 7]
```diff
@@ -0,0 +1,540 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ZCgWzNBkaQLZ",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Model Training\n",
+        "## Setting up Data within the config\n",
+        "\n",
+        "Among other things, the config file contains dictionaries called dataset, train_ds and validation_ds. These are configurations used to setup the Dataset and DataLoaders of the corresponding config.\n",
```
Comment (yzhang123) [inline tutorials/nlp/GLUE_Benchmark_tutorial.ipynb:255]: highlight dataset, train_ds, validation_ds

File: tutorials/nlp/GLUE_Benchmark_tutorial.ipynb [hunk 8]
```diff
@@ -0,0 +1,540 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8tjLhUvL_o7_",
+        "colab_type": "text"
+      },
+      "source": [
+        "Before initializing the model, we might want to modify some of the model configs. For example, we might want to modify the pretrained BERT model, for example use Megatron-LM BERT or AlBERT model:"
```
Comment (yzhang123) [inline tutorials/nlp/GLUE_Benchmark_tutorial.ipynb:374]: A"L"BERT. Please add links to Albert and megatron

Comment (ekmb) [reply tutorials/nlp/GLUE_Benchmark_tutorial.ipynb]: done

**Review by yzhang123 — APPROVED** [review]

**Pull request merged**